frontends/aiger/aigerparse.cc

```diff
@@ -440,16 +444,18 @@ void AigerReader::parse_xaiger(const dict<int,IdString> &box_lookup)
 			}
 			else if (c == 'h') {
 				f.ignore(sizeof(uint32_t));
+			#ifndef NDEBUG
```

Comment:
`parse_xaiger_literal` has an `istream::read` in it, so disabling it in NDEBUG looks to be incorrect?

Reply:
Thanks! I misread that code.. Should be fixed now.